src/hooks/searchEngines/useWfsSearchEngine.ts

```diff
@@ -81,6 +81,26 @@ export const useWfsSearchEngine = () => {
     return feature.getId();
   }, [replaceTemplates]);
 
+  const applyAttributesToFeature = (
+    feature: OlFeature
+  ): void => {
+    const blacklistedAttributes = [
+      'category',
+      'id',
+      'featureType',
+      'geometry',
+      'search'
```

Comment:
Where is this blacklist coming from? What happens if a feature actually contains the `category` attribute?

Reply:
I have used the blacklist from the https://github.com/terrestris/shogun-gis-client/blame/next/src/hooks/searchEngines/useWfsSearchEngine.ts#L59 as a reference.  
Should we modify this list in any way?

Reply:
Ah, obviously I introduced it ;-)

Yes, I would suggest to simplify this code snippet to filter out just the geometry. Since this PR allows to define the properties visible in the UI, this shouldn't be a problem, right?

Reply:
Right, as they can be configured now, I have removed all other fields but `geometry`.